### PR TITLE
.

### PR DESCRIPTION
**Pull request by DEAD-VICKY**
due to abuse i will not be finishing this

### CODE REVIEW

**Comment by DimWhat** [comment]
holy hell I haven't even look at everything and I'm already apalled by this. Hell no.

**Comment by Zonespace27** [comment]
Nice verified tags. Have you _actually_ tested any of this?

File: code/datums/commodity.dm [hunk 1]
```diff
@@ -2882,3 +2882,821 @@
...
+	
+/datum/commodity/blackmarket/wrestler/videocard
+	comname = "Advanced Videocard"
+	comtype = /obj/item/peripheral/videocard
+	desc_sell = "JUICE UP your computer's MUSCLES with one of these bad boys. Guaranteed to get you MORE FRAMES of HYPER ROBUST ACTION!"
+	price = 8000
+	upperfluc = 3000
+	lowerfluc = -3000
+	
+/datum/commodity/blackmarket/wrestler/laptop
```
Comment (DimWhat) [inline code/datums/commodity.dm:3479]: You can already buy these.

File: code/datums/commodity.dm [hunk 2]
```diff
@@ -2882,3 +2882,821 @@
...
+	lowerfluc = -50
+/datum/commodity/blackmarket/wrestler/chompskey
+	comname = "Chomp Key"
+	comtype = /obj/item/device/key/chompskey
+	desc_sell = "I need one of these URGENTLY!"
+	price = 20000
+	baseprice = 20000
+	upperfluc = 2000
+	lowerfluc = -2000
+/datum/commodity/blackmarket/wrestler/hairballkey
```
Comment (DimWhat) [inline code/datums/commodity.dm:3590]: Unable to even be found.

File: code/datums/commodity.dm [hunk 3]
```diff
@@ -2882,3 +2882,821 @@
...
+	//would like to have the price higher but alas it's avaliable in a QM crate and valued at about 7.5k there
+	comname = "NASSA space suit"
+	comtype = /obj/item/clothing/under/gimmick/blackstronaut
+	desc_sell = "I'd LOVE to get my hands on a couple sets of that prototype jumpsuit. WRESTLING IN THE COLD VACCUM OF SPACE! I'll pay a BUNCH for any you find in good condition!"
+	price = 6200
+	upperfluc = 800
+	lowerfluc = -800
+/datum/commodity/blackmarket/wrestler/uplink_telecrystal
+	//this is the bad guy tokens, and it's a lowball offer. this is primarily for using telescrystals to bribe crew members (who have no legitimate use for TCs)
+	comname = "Pure Telecrystal Token"
```
Comment (DimWhat) [inline code/datums/commodity.dm:3643]: Would prefer to use the name of whatever the traitor points are that round.

File: code/datums/commodity.dm [hunk 4]
```diff
@@ -2882,3 +2882,821 @@
...
+	comtype = /obj/item/gun/energy/glitch_gun
+	desc_sell = "You hear about the fabled Glitch Gun? It's said to be able to shoot bullets though code itself. If you find it, SELL IT TO ME!"
+	price = 1620000
+	upperfluc = 1200
+	lowerfluc = -1200
+/datum/commodity/blackmarket/wrestler/capsuit
+	comname = "Captain's Space Suit"
+	comtype = /obj/item/clothing/suit/space/captain
+	desc_sell = "The old style green space suits NT used to issue to their captains are hella valuable, but unfortunately also hella rare. Hey, you're a Traser, aren't ya? Could you get your captain's space suit to me?"
+	price = 605000
```
Comment (DimWhat) [inline code/datums/commodity.dm:3676]: Way too much.

File: code/datums/commodity.dm [hunk 5]
```diff
@@ -2882,3 +2882,821 @@
...
+	lowerfluc = -100000
+/datum/commodity/blackmarket/wrestler/magic_sandal
+	//these things are everywhere
+	comname = "Magic Sandals"
+	comtype = /obj/item/clothing/shoes/sandal
+	desc_sell = "That said, those knock offs are DAMN WELL MADE! I'll buy those for a fair price!"
+	price = 6800
+	upperfluc = 1200
+	lowerfluc = -1200
+/datum/commodity/blackmarket/wrestler/glitch_gun
```
Comment (DimWhat) [inline code/datums/commodity.dm:3664]: Not able to be gotten at all.

File: code/modules/economy/trader.dm [hunk 6]
```diff
@@ -1436,3 +1436,388 @@
...
+//  ███   ███   ███   ███    ███    ███  ▐███         ███         ███    ███ ███     ███  ███   ███   ███   ███    ███ ███   ███  //
+//  ███   ███   ███   ███    ███    ▀███▄███▀         ███         ███    ███ ███     ███  ███   ███   ███   ███    ███ ███   ███  //
+//  ███   ███   ███ ▀███████████    ████▀██▄          ███       ▀███████████ ███     ███  ███   ███   ███ ▀███████████ ███   ███  //
+//  ███   ███   ███   ███    ███   ▐███  ▀███         ███         ███    ███ ███     ███  ███   ███   ███   ███    ███ ███   ███  //
+//  ███   ███   ███   ███    ███  ▄███     ███▄       ███▌    ▄   ███    ███ ███ ▄█▄ ███  ███   ███   ███   ███    ███ ███   ███  //
+//   ▀█   ███   █▀    ███    █▀  ████       ███▄      █████▄▄██   ███    █▀   ▀███▀███▀    ▀█   ███   █▀    ███    █▀   ▀█   █▀   //
+//                                                   ▀                                                                            //
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1464]: Stop with your big ascii art

File: code/modules/economy/trader.dm [hunk 7]
```diff
@@ -1436,3 +1436,388 @@
...
+	New()
+		..()
+
+//////////////////////////////////////////////////////////////
+///███████ ███████ ██      ██      ██ ███    ██  ██████     //
+///██      ██      ██      ██      ██ ████   ██ ██       ██ //
+///███████ █████   ██      ██      ██ ██ ██  ██ ██   ███    //
+///     ██ ██      ██      ██      ██ ██  ██ ██ ██    ██ ██ //
+///███████ ███████ ███████ ███████ ██ ██   ████  ██████     //
+//////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1485]: Stop.

File: code/modules/economy/trader.dm [hunk 8]
```diff
@@ -1436,3 +1436,388 @@
...
+		src.goods_sell += new /datum/commodity/blackmarket/surplus/handcuffs(src)
+		src.goods_sell += new /datum/commodity/blackmarket/surplus/riot_grenade(src)
+
+/////////////////////////////////////////////////////////
+///██████  ██    ██ ██    ██ ██ ███    ██  ██████     ///
+///██   ██ ██    ██  ██  ██  ██ ████   ██ ██       ██ ///
+///██████  ██    ██   ████   ██ ██ ██  ██ ██   ███    ///
+///██   ██ ██    ██    ██    ██ ██  ██ ██ ██    ██ ██ ///
+///██████   ██████     ██    ██ ██   ████  ██████     ///
+/////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1508]: Stop

File: code/modules/economy/trader.dm [hunk 9]
```diff
@@ -1436,3 +1436,388 @@
...
+		src.goods_buy += new /datum/commodity/blackmarket/surplus/powercell(src)
+		src.goods_buy += new /datum/commodity/blackmarket/surplus/customcell(src)
+		
+//////////////////////////////////////////////////////////////////////
+///██████  ██  █████  ██       ██████   ██████  ██    ██ ███████    //
+///██   ██ ██ ██   ██ ██      ██    ██ ██       ██    ██ ██      ██ //
+///██   ██ ██ ███████ ██      ██    ██ ██   ███ ██    ██ █████      //
+///██   ██ ██ ██   ██ ██      ██    ██ ██    ██ ██    ██ ██      ██ //
+///██████  ██ ██   ██ ███████  ██████   ██████   ██████  ███████    //
+//////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1524]: Stop.

File: code/modules/economy/trader.dm [hunk 10]
```diff
@@ -1436,3 +1436,388 @@
...
+//▐░█▄▄▄▄▄▄▄▄▄ ▐░▌       ▐░▌▐░█▄▄▄▄▄▄▄█░▌      ▐░▌    ▐░█▄▄▄▄▄▄▄▄▄ ▐░▌               ▐░▌               ▐░░▌         ▐░░▌    //
+//▐░░░░░░░░░░░▌▐░▌       ▐░▌▐░░░░░░░░░░▌       ▐░▌    ▐░░░░░░░░░░░▌▐░▌               ▐░▌               ▐░░▌         ▐░░▌    //
+// ▀▀▀▀▀▀▀▀▀█░▌▐░▌       ▐░▌▐░█▀▀▀▀▀▀▀█░▌      ▐░▌    ▐░█▀▀▀▀▀▀▀▀▀ ▐░▌               ▐░▌               ▐░░▌         ▐░░▌    //
+//          ▐░▌▐░▌       ▐░▌▐░▌       ▐░▌      ▐░▌    ▐░▌          ▐░▌               ▐░▌               ▐░░▌         ▐░░▌    //
+// ▄▄▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄█░▌ ▄▄▄▄▄█░▌    ▐░█▄▄▄▄▄▄▄▄▄ ▐░█▄▄▄▄▄▄▄▄▄      ▐░▌           ▄▄▄▄█░░█▄▄▄  ▄▄▄▄█░░█▄▄▄ //
+//▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░▌ ▐░░░░░░░▌    ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌     ▐░▌          ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌//
+// ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀   ▀▀▀▀▀▀▀      ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀       ▀            ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀ //
+//                                                                                                                          //
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1573]: Stop.

File: code/modules/economy/trader.dm [hunk 11]
```diff
@@ -1436,3 +1436,388 @@
...
+	New()
+		..()
+
+//////////////////////////////////////////////////////////////
+///███████ ███████ ██      ██      ██ ███    ██  ██████     //
+///██      ██      ██      ██      ██ ████   ██ ██       ██ //
+///███████ █████   ██      ██      ██ ██ ██  ██ ██   ███    //
+///     ██ ██      ██      ██      ██ ██  ██ ██ ██    ██ ██ //
+///███████ ███████ ███████ ███████ ██ ██   ████  ██████     //
+//////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1595]: Stop

File: code/modules/economy/trader.dm [hunk 12]
```diff
@@ -1436,3 +1436,388 @@
...
+		src.goods_sell += new /datum/commodity/blackmarket/blob/chitin(src)/datum/commodity/blackmarket/blob/head
+		src.goods_sell += new /datum/commodity/blackmarket/blob/head(src)
+		
+/////////////////////////////////////////////////////////
+///██████  ██    ██ ██    ██ ██ ███    ██  ██████     ///
+///██   ██ ██    ██  ██  ██  ██ ████   ██ ██       ██ ///
+///██████  ██    ██   ████   ██ ██ ██  ██ ██   ███    ///
+///██   ██ ██    ██    ██    ██ ██  ██ ██ ██    ██ ██ ///
+///██████   ██████     ██    ██ ██   ████  ██████     ///
+/////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1606]: Stop.

File: code/modules/economy/trader.dm [hunk 13]
```diff
@@ -1436,3 +1436,388 @@
...
+
+		src.goods_buy += new /datum/commodity/blackmarket/blob/heisenbee(src)
+
+//////////////////////////////////////////////////////////////////////
+///██████  ██  █████  ██       ██████   ██████  ██    ██ ███████    //
+///██   ██ ██ ██   ██ ██      ██    ██ ██       ██    ██ ██      ██ //
+///██   ██ ██ ███████ ██      ██    ██ ██   ███ ██    ██ █████      //
+///██   ██ ██ ██   ██ ██      ██    ██ ██    ██ ██    ██ ██      ██ //
+///██████  ██ ██   ██ ███████  ██████   ██████   ██████  ███████    //
+//////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1616]: Stop

File: code/modules/economy/trader.dm [hunk 14]
```diff
@@ -1436,3 +1436,388 @@
...
+//      ▒██    ▒ ▓██░ ██▒▒████▄    ▒██▀ ██▌ ▒██  ██▒      ▒██   ▒████▄     ██ ▀█   █ ▓█   ▀     //
+//      ░ ▓██▄   ▒██▀▀██░▒██  ▀█▄  ░██   █▌  ▒██ ██░      ░██   ▒██  ▀█▄  ▓██  ▀█ ██▒▒███       //
+//       ▒   ██▒░▓█ ░██ ░██▄▄▄▄██ ░▓█▄   ▌  ░ ▐██▓░   ▓██▄██▓  ░██▄▄▄▄██ ▓██▒  ▐▌██▒▒▓█  ▄      //
+//     ▒██████▒▒░▓█▒░██▓ ▓█   ▓██▒░▒████▓   ░ ██▒▓░    ▓███▒    ▓█   ▓██▒▒██░   ▓██░░▒████▒     //
+//      ▒ ▒▓▒ ▒ ░ ▒ ░░▒░▒ ▒▒   ▓▒█░ ▒▒▓  ▒    ██▒▒▒     ▒▓▒▒░    ▒▒   ▓▒█░░ ▒░   ▒ ▒ ░░ ▒░ ░    //
+//      ░ ░▒  ░ ░ ▒ ░▒░ ░  ▒   ▒▒ ░ ░ ▒  ▒  ▓██ ░▒░     ▒ ░▒░     ▒   ▒▒ ░░ ░░   ░ ▒░ ░ ░  ░    //
+//      ░  ░  ░   ░  ░░ ░  ░   ▒    ░ ░  ░  ▒ ▒ ░░      ░ ░ ░     ░   ▒      ░   ░ ░    ░       //
+//            ░   ░  ░  ░      ░  ░   ░     ░ ░         ░   ░         ░  ░         ░    ░  ░    //
+//                                  ░       ░ ░                                                 //
+//////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1653]: Stop

File: code/modules/economy/trader.dm [hunk 15]
```diff
@@ -1436,3 +1436,388 @@
...
+	New()
+		..()
+
+//////////////////////////////////////////////////////////////
+///███████ ███████ ██      ██      ██ ███    ██  ██████     //
+///██      ██      ██      ██      ██ ████   ██ ██       ██ //
+///███████ █████   ██      ██      ██ ██ ██  ██ ██   ███    //
+///     ██ ██      ██      ██      ██ ██  ██ ██ ██    ██ ██ //
+///███████ ███████ ███████ ███████ ██ ██   ████  ██████     //
+//////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1674]: Stop

File: code/modules/economy/trader.dm [hunk 16]
```diff
@@ -1436,3 +1436,388 @@
...
+		src.goods_sell += new /datum/commodity/blackmarket/shady/balaclava(src)
+		src.goods_sell += new /datum/commodity/blackmarket/shady/brick(src)
+
+/////////////////////////////////////////////////////////
+///██████  ██    ██ ██    ██ ██ ███    ██  ██████     ///
+///██   ██ ██    ██  ██  ██  ██ ████   ██ ██       ██ ///
+///██████  ██    ██   ████   ██ ██ ██  ██ ██   ███    ///
+///██   ██ ██    ██    ██    ██ ██  ██ ██ ██    ██ ██ ///
+///██████   ██████     ██    ██ ██   ████  ██████     ///
+/////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1691]: Stop

File: code/modules/economy/trader.dm [hunk 17]
```diff
@@ -1436,3 +1436,388 @@
...
+		src.goods_buy += new /datum/commodity/blackmarket/shady/crowbar(src)
+		src.goods_buy += new /datum/commodity/blackmarket/shady/wirecutters(src)
+
+//////////////////////////////////////////////////////////////////////
+///██████  ██  █████  ██       ██████   ██████  ██    ██ ███████    //
+///██   ██ ██ ██   ██ ██      ██    ██ ██       ██    ██ ██      ██ //
+///██   ██ ██ ███████ ██      ██    ██ ██   ███ ██    ██ █████      //
+///██   ██ ██ ██   ██ ██      ██    ██ ██    ██ ██    ██ ██      ██ //
+///██████  ██ ██   ██ ███████  ██████   ██████   ██████  ███████    //
+//////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1706]: Stop

File: code/modules/economy/trader.dm [hunk 18]
```diff
@@ -1436,3 +1436,388 @@
...
+		pickupdialoguefailure = "Some kind of error here? Says you don't have anything in your cart?"
+
+//////////////////////////////////////////////////////////////////////////////////////////
+//	███╗   ███╗ █████╗ ██████╗     ██╗      █████╗ ██████╗ ██████╗ ██╗   ██╗	//
+//	████╗ ████║██╔══██╗██╔══██╗    ██║     ██╔══██╗██╔══██╗██╔══██╗╚██╗ ██╔╝	//
+//	██╔████╔██║███████║██║  ██║    ██║     ███████║██████╔╝██████╔╝ ╚████╔╝ 	//
+//	██║╚██╔╝██║██╔══██║██║  ██║    ██║     ██╔══██║██╔══██╗██╔══██╗  ╚██╔╝  	//
+//	██║ ╚═╝ ██║██║  ██║██████╔╝    ███████╗██║  ██║██║  ██║██║  ██║   ██║   	//
+//	╚═╝     ╚═╝╚═╝  ╚═╝╚═════╝     ╚══════╝╚═╝  ╚═╝╚═╝  ╚═╝╚═╝  ╚═╝   ╚═╝           //
+//////////////////////////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1738]: Stop

File: code/modules/economy/trader.dm [hunk 19]
```diff
@@ -1436,3 +1436,388 @@
...
+	New()
+		..()
+		
+//////////////////////////////////////////////////////////////
+///███████ ███████ ██      ██      ██ ███    ██  ██████     //
+///██      ██      ██      ██      ██ ████   ██ ██       ██ //
+///███████ █████   ██      ██      ██ ██ ██  ██ ██   ███    //
+///     ██ ██      ██      ██      ██ ██  ██ ██ ██    ██ ██ //
+///███████ ███████ ███████ ███████ ██ ██   ████  ██████     //
+//////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1758]: Stop

File: code/modules/economy/trader.dm [hunk 20]
```diff
@@ -1436,3 +1436,388 @@
...
+		src.goods_sell += new /datum/commodity/blackmarket/
+
+
+/////////////////////////////////////////////////////////
+///██████  ██    ██ ██    ██ ██ ███    ██  ██████     ///
+///██   ██ ██    ██  ██  ██  ██ ████   ██ ██       ██ ///
+///██████  ██    ██   ████   ██ ██ ██  ██ ██   ███    ///
+///██   ██ ██    ██    ██    ██ ██  ██ ██ ██    ██ ██ ///
+///██████   ██████     ██    ██ ██   ████  ██████     ///
+/////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1777]: Stop

File: code/modules/economy/trader.dm [hunk 21]
```diff
@@ -1436,3 +1436,388 @@
...
+
+
+		src.goods_sell += new /datum/commodity/blackmarket/
+		src.goods_sell += new /datum/commodity/blackmarket/wrestler/videocard
+		src.goods_sell += new /datum/commodity/blackmarket/wrestler/
+		src.goods_sell += new /datum/commodity/blackmarket/wrestler/
+		src.goods_sell += new /datum/commodity/blackmarket/cowboyboots
+		src.goods_sell += new /datum/commodity/blackmarket/goldfishcracker
+		src.goods_sell += new /datum/commodity/blackmarket/
+		src.goods_sell += new /datum/commodity/blackmarket/
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1768]: Some of these paths don't do anything

File: code/modules/economy/trader.dm [hunk 22]
```diff
@@ -1436,3 +1436,388 @@
...
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/nassa
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/sacred_sandal
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/magic_sandal
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/teapot
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/magic_sandal
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/shoecrafts
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/
+		src.goods_buy += new /datum/commodity/blackmarket/wrestler/
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1789]: Again, blank paths.

File: code/modules/economy/trader.dm [hunk 23]
```diff
@@ -1436,3 +1436,388 @@
...
+
+//////////////////////////////////////////////////////////////////////
+///██████  ██  █████  ██       ██████   ██████  ██    ██ ███████    //
+///██   ██ ██ ██   ██ ██      ██    ██ ██       ██    ██ ██      ██ //
+///██   ██ ██ ███████ ██      ██    ██ ██   ███ ██    ██ █████      //
+///██   ██ ██ ██   ██ ██      ██    ██ ██    ██ ██    ██ ██      ██ //
+///██████  ██ ██   ██ ███████  ██████   ██████   ██████  ███████    //
+//////////////////////////////////////////////////////////////////////
```
Comment (DimWhat) [inline code/modules/economy/trader.dm:1797]: Stop.

**Review by DimWhat — COMMENTED** [review]
Okay so this is only some of the stuff that I've seen. This needs a LOT of work and I'm not even talking about the mere idea of a black market.

**Comment by Zonespace28** [comment]
Hey there! Just for pure transparency, my main account, @Zonespace27, was blocked by @DEAD-VICKY, making me unable to comment on this PR, following
https://github.com/goonstation/goonstation/pull/7126#issuecomment-1017996494 this comment,
which followed this https://forum.ss13.co/showthread.php?tid=17721 forum thread, and I fully expect this account to get blocked, too!

File: code/datums/commodity.dm [hunk 24]
```diff
@@ -2882,3 +2882,821 @@
...
+	price = 4000
+	upperfluc = 2000
+	lowerfluc = -2000
+	New()
+		..()
+		switch (lowertext( time2text(world.realtime, "Day") ))
+			if ("monday")
+				comtype = /obj/item/clothing/under/rank/security
+			if ("tuesday")
+				comtype = /obj/item/clothing/under/chameleon
```
[the remark this answers is not in the document]
Comment (ZeWaka) [reply code/datums/commodity.dm:2970]: ^

File: code/datums/commodity.dm [hunk 25]
```diff
@@ -2882,3 +2882,821 @@
...
+	price = 8000
+	upperfluc = 2000
+	lowerfluc = -2000
+		New()
+		..()
+		switch (lowertext( time2text(world.realtime, "Day") ))
+			if ("monday")
+				comtype = /obj/item/clothing/suit/armor/vest
+			if ("tuesday")
+				comtype = /obj/item/clothing/suit/armor/turd
```
Comment (ZeWaka) [inline code/datums/commodity.dm:2998]: no

File: code/datums/commodity.dm [hunk 26]
```diff
@@ -2882,3 +2882,821 @@
...
+			if ("wednesday")
+				comtype = /obj/item/clothing/suit/fire/armored
+			if ("thursday")
+				comtype = /obj/item/clothing/suit/armor/EOD
+			if ("friday")
+				comtype = /obj/item/clothing/suit/armor/vest
+			if ("saturday")
+				comtype = /obj/item/clothing/suit/armor/tdome/red
```
Comment (ZeWaka) [inline code/datums/commodity.dm:3006]: no

File: code/datums/commodity.dm [hunk 27]
```diff
@@ -2882,3 +2882,821 @@
...
+			if ("monday")
+				comtype = /obj/item/clothing/shoes/bootsblk
+			if ("tuesday")
+				comtype = /obj/item/clothing/shoes/detective
+			if ("wednesday")
+				comtype = /obj/item/clothing/shoes/swat
+			if ("thursday")
+				comtype = /obj/item/clothing/shoes/bootsblk
+			if ("friday")
+				comtype = /obj/item/clothing/shoes/swat/noslip
```
[the remark this answers is not in the document]
Comment (ZeWaka) [reply code/datums/commodity.dm:3037]: ^

File: code/datums/commodity.dm [hunk 28]
```diff
@@ -2882,3 +2882,821 @@
...
+	upperfluc = 40000
+	lowerfluc = -40000
+/datum/commodity/blackmarket/surplus/lasergun
+	comname = "Laser Gun"
+	comtype = /obj/item/gun/energy/laser_gun
+	desc =  = "Need some of those older NT laser guns. The obsolete ones with the round bodies. The parts are invaluable."
+	price = 12000
+	upperfluc = 3000
+	lowerfluc = -3000
+/datum/commodity/blackmarket/surplus/phaser
```
[the remark this answers is not in the document]
Comment (ZeWaka) [reply code/datums/commodity.dm:3144]: ^

**Review by ZeWaka — CHANGES_REQUESTED** [review]
1. incorrect changelog
2. no giant word ascii art headers
3. this doesn't even compile bro
4. follow the contributor guidelines
5. this PR isn't labeled and i won't look at it again until it is

File: code/datums/commodity.dm [hunk 29]
```diff
@@ -2882,3 +2882,821 @@
...
+	lowerfluc = -30000
+/datum/commodity/blackmarket/wrestler/shoecrafts
+	//space sweatshop 13
+	comname = "Custom Shoes"
+	comtype = /obj/item/clothing/shoes/crafted
+	desc_sell = "And any shoes YOU HAND CRAFT, I WILL BUY!"
+	price = 800
+	upperfluc = 600
+	lowerfluc = -600
+/datum/commodity/blackmarket/wrestler/katana_nukeop
```
Comment (Eagletanker) [inline code/datums/commodity.dm:3695]: (not a problem with code just want to leave a note)
Nuke Op's dont last long enough to reach the time when this trader would become active according to your post on the forums. I'd like to clarify too, that with all my stuff, I'm not a dev. I'm some dumbass who doesnt know how to get out of review mode in git and ended up in it on accident.

**Pull request closed without merging**